### DISCUSSION TITLE
What 📖 holds the Guinness World Record for fastest selling book?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#985] [Galaxy Trivia] [answered]
**Bonus:** When It was first published?

**Comment by Sriansh-raj** [comment]
Prince Harry's memoir "Spare" holds the Guinness World Record for the fastest-selling non-fiction book of all time, having sold 1.43 million copies in just 24 hours in the UK, US, and Canada.

It was first published on January 10, 2023.

**Comment by peterbaccaro** [comment]
**Harry Potter and The Deathly Hallows** — the final novel of J.K. Rowling's series — currently holds the Guinness World Record for the fastest selling book of all time. Within 24 hours the book had sold well over 11 million copies worldwide.

Harry Potter and the Deathly Hallows by J.K. Rowling was **first published on July 21, 2007**.

**Comment by thuongtruong1009** [comment, marked as answer]
The Guinness World Record for the fastest-selling book is held by the novel "Harry Potter and the Deathly Hallows" by J.K. Rowling. It was first published on July 21, 2007. The book is the seventh and final installment in the Harry Potter series, and it sold a staggering 11 million copies within the first 24 hours of its release in the United Kingdom and the United States combined. The record-breaking sales highlight the immense popularity and anticipation surrounding the conclusion of the beloved Harry Potter series.
